### PR TITLE
Tips on Running Async Remote Hackathons Page

### PR DESCRIPTION
**Pull request by Joelkw**
Documenting the learnings + processes of the developer insights hackathon FY22Q3, to support future remote async hackathons. 

### Open questions

- Should we make this specifically a product/eng hackathon page, or try to expand it to incorporate info from the very recent bizops and CE hackathons? (See the blog hackathon page already linked). 
   - I can see arguments for making it specific to prod+eng, but I personally would love to see more cross-collaborative hackathons (eg, CEs paired with eng, or a "slice" of every department of the company doing a hackathon, etc) so I may encourage folks to restructure/combine information into this one central page and then link out to individual, specific pages as necessary.

### CODE REVIEW

**Comment by Joelkw** [comment]
(I haven't done a typo pass yet but wanted to get this up already; will do that before merging.)

**Comment by Joelkw** [comment]
cc @benvenker

**Comment by benvenker** [comment]
oh awesome, thanks @Joelkw!

File: content/company/remote/remote_hackathons.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# Remote hackathons
+
+This page describes processes and tips for holding remote hackathons. Everything on this page should be taken as a suggestion – you can decide whether or not something should apply to an individual future hackathon.
+
+See also [Blog Hackathon](../../marketing/content/editorial/blog_hackathon) which describes how the content team has run hackathons.
```
Comment (jasonyavorsky) [inline content/company/remote/remote_hackathons.md]: I suspect that this is the fix

```suggestion
See also [Blog Hackathon](../../marketing/content/editorial/blog_hackathon.md) which describes how the content team has run hackathons.
```

**Comment by jasonyavorsky** [comment]
This is great @Joelkw ! I'd just merge it once you have it building without errors. I suspect that you're the company expert on running these these days, and anyone with follow-on ideas can add them as subsequent iterations.

**Pull request merged**